src/lando/ui/legacy/revisions.py

```diff
@@ -21,6 +21,49 @@
 # revisions.before_request(set_last_local_referrer)
 
 
+class Uplift(LandoView):
+    @force_auth_refresh
+    def post(self, request: HttpRequest) -> HttpResponse:
+        """Process the uplift request submission."""
+        uplift_request_form = UpliftRequestForm(request.POST)
+        errors = []
+
+        if not request.user.is_authenticated:
+            raise PermissionError()
+
+        if uplift_request_form.is_valid() and not errors:
```

Comment:
`errors` will always be empty here, as it only gets set below on line 53. I'd suggest moving the error management LL52-64 up before this, and return early, and keep the happy case as the continuation of the method.